src/app/static/src/tests/components/errors.test.ts

```diff
@@ -21,25 +20,25 @@ describe("Errors component", () => {
         })
     };
 
-    const propsData = {title: "TestApp"};
+    const props = {title: "TestApp"};
 
     it("renders as expected", () => {
         const store = createStore([
             {error: "First error", detail: null},
             {error: "Second error", detail: "Error detail"}
         ]);
-        const wrapper = shallowMount(Errors, {propsData, store, localVue});
+        const wrapper = shallowMount(Errors, {props, global: {plugins: [store]}});
 
         const paras = wrapper.findAll("p");
         expect(paras.length).toBe(2);
-        expect(paras.at(0).text()).toBe("First error");
-        expect(paras.at(1).text()).toBe("Error detail");
+        expect(paras[0].text()).toBe("First error");
+        expect(paras[1].text()).toBe("Error detail");
     });
 
     it("renders nothing when no errors", () => {
         const store = createStore([]);
-        const wrapper = shallowMount(Errors, {propsData, store, localVue});
-        expect(wrapper.html()).toBeFalsy();
+        const wrapper = shallowMount(Errors, {props, global: {plugins: [store]}});
+        expect(wrapper.html()).toBe("<!--v-if-->");
```

Comment:
yes, so this is the same as empty wrapper basically so nothing shown because vue uses `<!--v-if-->` to keep track of where the `v-if`s are (conditional rendering) so if the html is just `<!--v-if-->` that means the `hasErrors` variable in `Errors.vue` is false. If it was true, Vue would replace the `<!--v-if-->` comment with our template